custom/intrahealth/sqldata.py

```diff
@@ -944,10 +946,9 @@ class VisiteDeLOperateurPerProductDataSource(YeksiSqlData):
 
     @property
     def filters(self):
-        filters = [
-            BETWEEN("real_date_repeat", "startdate", "enddate"),
-            ContainsFilter("select_programs", "program")
-        ]
+        filters = [BETWEEN("real_date_repeat", "startdate", "enddate")]
+        if self.config.get('program'):
```

Comment:
we really only need to check for `None` so it is probably better as
`if self.config.get('program') is not None` or even `if self.config['program'] is not None` since we know it will exist but its not blocking

Reply:
@calellowitz I think this can be an empty string if we choose All option.